ext/standard/tests/array/gh18480.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+GH-18480 (array_splice overflow with large offset / length values)
+--FILE--
+<?php
+$a = [PHP_INT_MAX];
+$offset = PHP_INT_MAX;
+var_dump(array_splice($a,$offset, PHP_INT_MAX));
+$offset = PHP_INT_MIN;
+var_dump(array_splice($a,$offset, PHP_INT_MIN));
```

Comment:
Tests should always end with `?>`

Reply:
Also you didn't test every combination of PHP_INT_MIN/PHP_INT_MAX with offset/length, you should have 4 tests in total. You should also reassign the `$a` array before every test so they don't influence each other.

Reply:
I think more if we try to test non packed arrays too.